CI/physmon/workflows/physmon_trackfinding_1muon.py

```diff
@@ -128,10 +128,11 @@ def run_ckf_tracking(label, seeding):
                 1 * u.mm,
                 1 * u.degree,
                 1 * u.degree,
-                0.1 * u.e / u.GeV,
+                0 * u.e / u.GeV,
```

Comment:
This means that initial Q/p sigma is 0 if I understand correctly the logic, right?